salt/metalk8s/orchestrate/downgrade/precheck.sls

```diff
@@ -16,6 +18,13 @@ Correct saltenv "{{ saltenv }}" for downgrade to "{{ dest_version }}":
 
 {%- endif %}
 
+{%- if not defaults.downgrade_supported and not pillar.force_downgrade | default(False) %}
```

Comment:
Maybe we want this pillar entry below the `metalk8s` one ? `pillar.metalk8s.force_downgrade` or even something like `pillar.metalk8s.downgrade.force` ?

Reply:
I'd put everything under `metalk8s.downgrade`, either using both `.supported` and `.force`, or with a single key, maybe naming it `.enabled`?

Reply:
(No strong opinion about the place for `force_downgrade` but maybe since it's only a pillar key that will be passed to orchestrate I would say to be consistent `metalk8s.orchestrate.force_downgrade` (or `metalk8s.orchestrate.downgrade.force` as you want))
Also here the check is wrong I think, because we can downgrade from 2.6.1 to 2.6.0 for example. IMO you should add a check about the destination version here